operations/mimir-mixin/alerts/alerts.libsonnet

```diff
@@ -29,30 +55,8 @@ local utils = import 'mixin-utils/utils.libsonnet';
             message: '%(product)s cluster %(alert_aggregation_variables)s has {{ printf "%%f" $value }} unhealthy ingester(s).' % $._config,
           },
         },
-        {
-          alert: $.alertName('RequestErrors'),
-          // Note if alert_aggregation_labels is "job", this will repeat the label. But
-          // prometheus seems to tolerate that.
-          expr: |||
-            100 * sum by (%(group_by)s, %(job_label)s, route) (rate(cortex_request_duration_seconds_count{status_code=~"5..",route!~"%(excluded_routes)s"}[1m]))
-              /
-            sum by (%(group_by)s, %(job_label)s, route) (rate(cortex_request_duration_seconds_count{route!~"%(excluded_routes)s"}[1m]))
-              > 1
-          ||| % {
-            group_by: $._config.alert_aggregation_labels,
-            job_label: $._config.per_job_label,
-            excluded_routes: std.join('|', ['ready'] + $._config.alert_excluded_routes),
-          },
-          'for': '15m',
-          labels: {
-            severity: 'critical',
-          },
-          annotations: {
-            message: |||
-              The route {{ $labels.route }} in %(alert_aggregation_variables)s is experiencing {{ printf "%%.2f" $value }}%% errors.
-            ||| % $._config,
-          },
-        },
+        requestErrorsAlert('critical', 'route!~"%s"' % std.join('|', ['ready'] + $._config.alert_excluded_routes + $._config.alert_non_critical_routes)),
+        requestErrorsAlert('warning', 'route="%s"' % std.join('|', $._config.alert_non_critical_routes)),
```

Comment:
Should this also exclude the 'excluded_routes'?

Reply:
The matcher for warning alerts is an equality (`=`) matcher, so it'll only fire for non-critical routes.

(It should be a regex equality matcher though... I'll fix it now.)